push.cmd

```diff
@@ -0,0 +1,30 @@
+@ECHO OFF
```

Comment:
it looks like this is an extra file

Reply:
Yea, sorry this is the command that I used to push to remote repository. It can be removed.

Reply:
remove it then :)

Reply:
Remove the this file already, resolving this conversation.